cli/src/main.rs

```diff
@@ -223,7 +227,7 @@ fn setup_logger() -> anyhow::Result<()> {
 
 fn print_cli_start_info() {
     log::info!(
-        "Starting trunk-analytics-cli {} (git={}) rustc={}",
+        "Starting trunk flakytests {} (git={}) rustc={}",
```

Comment:
This won't be correct for existing users that have downloaded the release directly from GitHub.